lib/sprinkler_web/mqtt/handler.ex

```diff
@@ -15,10 +17,10 @@ defmodule SprinklerWeb.Mqtt.Handler do
     {:ok, state}
   end
 
-  def handle_message(["rs", _client_id, "telemetry"], _payload, state) do
+  def handle_message(["rs", _client_id, "telemetry"], payload, state) do
     # What should we do with sensor information?
-    # IO.inspect(client_id)
-    # IO.inspect(payload)
+    # In which format are we receiving the temp? Might have to change this later.
+    Phoenix.PubSub.broadcast(Sprinkler.PubSub, @telemetry, %{topic: @telemetry, temp: payload})
```

Comment:
It'd be nice to test that messages are being sent correctly

Reply:
We can use a shorthand here

```elixir
SprinklerWeb.Endpoint.broadcast(@ telemetry, "new_reading", %{temp: payload})
```

I don't know if it's a better approach though